sys/shell/commands/sc_cryptoauthlib.c

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright (C) 2020 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_shell_commands
+ * @{
+ *
+ * @file
+ * @brief       Shell commands for the cryptoauthlib module
+ *
+ * @author      Lena Boeckmann <lena.boeckmann@haw-hamburg.de>
+ *
+ * @}
+ */
+#include <stdio.h>
+#include <stdint.h>
+#include "cryptoauthlib.h"
+
+void get_bin(char *result, uint8_t byte)
+{
+    for (int i = 0; i < 8; i++) {
+        result[i] = (byte << i & 0x80 ? '1' : '0');
```

Comment:
mhm is this correct? You compare starting from highest bit? Doesn't this invert the order when printing?
Or is that intended?

I would do (found online https://codereview.stackexchange.com/questions/82303/byte-to-binary-conversion-function):

```
result[i] = '0' + ((byte >> i) & 1);
```

Reply:
I don't think this inverts the order. For example if I get a byte 0x50 I want to print this binary number: 01010000. This means I need to write them into the array msb first, right? 
So I compare with 0x80 to get the msb and shift left, to get the next bit.

If I shift right and compare with 1, I'd get the lsb first, which inverts the order.

Reply:
I just checked my output again, specifically the revision number and the lock bytes, as they are fixed values, they're not inverted.

Reply:
yeah, if you want msb first then yours is correct. That's why I asked, so it is intended - fine with me 😄 

I was think "network world", where packet headers are typically printed lsb first with bit 0 to 31 from left to right.